src/loadContent.js

```diff
@@ -120,12 +120,18 @@ export function loadContent() {
     return;
   }
 
+  if (hashIds) {
+    learnerId = await hashString(learnerId);
+    learnerName = await hashString(learnerName);
```

Comment:
Shouldn’t we just remove the learner name when we’re hashing IDs? The intent is to preserve anonymity, having a hashed learner name in addition to the hashed ID doesn’t bring much (I’d argue it adds confusion).

Reply:
Good point, for the moment Didask is waiting for both fields to be defined. I can make it optional wherever necessary, but the gain/effort ratio makes me think I shouldn't do it.

Reply:
And there are already some existing learner_name in the database, they will be hashed but will still exists